src/ai/backend/common/types.py

```diff
@@ -346,6 +346,8 @@ def check_typed_tuple(value: Tuple[Any, ...], types: Tuple[Type, ...]) -> Tuple:
 
 ResourceGroupID = NewType("ResourceGroupID", str)
 AgentId = NewType("AgentId", str)
+AGENTID_MANAGER = AgentId("manager")
+AGENTID_STORAGE = AgentId("storage")
 DeviceName = NewType("DeviceName", str)
```

Comment:
AgentId is a type meant to hold the ID of an agent.
I don't think the manager and the storage proxy should be considered agents, and using this kind of type is confusing.

Reply:
This is a temporary addition, and like you said, I'd like to change it to accept values like sourceId.
This is a temporary field to at least ensure the behavior now.